src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -230,7 +231,11 @@ export default class Mutations extends React.Component<
                         isPutativeDriver={
                             this.props.store.driverAnnotationSettings
                                 .driversAnnotated
-                                ? (m: AnnotatedMutation) => m.putativeDriver
+                                ? (m: AnnotatedMutation) =>
```

Comment:
too much logic here.  we need put this in a function and also parenthesize the terms properly || &&